lua/entities/gmod_wire_expression2/core/egpfunctions.lua

```diff
@@ -1120,7 +1083,7 @@ end
 
 --[[ currently does not work
 e2 function number wirelink:egpClearQueue()
-	if (!EGP:ValidEGP( this )) then return end
+	if (!EGP:ValidEGP( this )) then return 0 end
```

Comment:
Nothing wrong with fixing commented out code, if it's an obvious fix. That way it's less broken if someone uncomments it later.